iphone/TitaniumKit/TitaniumKit/Sources/API/TiWindowProxy.m

```diff
@@ -558,6 +558,13 @@ - (void)openOnUIThread:(NSArray *)args
           [theController setModalPresentationStyle:style];
         }
       }
+
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+      if ([TiUtils isIOSVersionOrGreater:@"13.0"]) {
+        isForceModalDisable = ![TiUtils boolValue:@"forceModal" properties:dict def:NO];
```

Comment:
The double negation here and in the following line is confusing. Consider changing the variable name so you don't have to negate the result twice, for example:

```suggestion
        forceModalBehavior = [TiUtils boolValue:@"forceModal" properties:dict def:NO];
```

Reply:
Updated.